src/App.tsx

```diff
@@ -96,6 +98,9 @@ const App: FC = () => {
           });
       }
     }
+    if (i18next.language.includes('en')) {
```

Comment:
trenger vi ikke gjøre det samme for 'no' ?

Reply:
/public/index.html har allerede satt <html lang="no"> som default.

Reply:
ah